packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -140,12 +140,20 @@ function CommentBox({
     <>
       <article className="flex flex-col items-stretch rounded-8 break-words-overflow typo-callout">
         <header className="flex items-center mb-2">
-          <ProfilePicture
-            size="large"
-            rounded="full"
-            user={{ image: authorImage, username: authorName }}
-            nativeLazyLoading
-          />
+          {isComment ? (
+            <ProfilePicture
+              size="large"
+              rounded="full"
```

Comment:
You can just make this line optional based on isComment, but is it the right evaluation? Not sure what it fires on.